src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -75,6 +75,10 @@ protected void signalFailureToDetectException(String... messages) {
         throw new RuntimeException("Expected exception not detected." + Arrays.toString(messages));
     }
 
+    protected void signalFailureToDetectAssertionError(String... messages) {
```

Comment:
Look at the existing method `signalFailureToDetectException`, your method is doing the same thing.